### ISSUE TITLE
Documentation not up to date for louvain_communities function.

### ISSUE THREAD

**Issue by pauldubois98** [networkx/networkx#6618]
The documentation of the function `louvain_communities` is not up to date.

### Current Behavior
The import in the code example is wrong:
```
import networkx as nx
from networkx.algorithms import community
G = nx.petersen_graph()
nx_comm.louvain.louvain_communities(G, seed=123)
```
gives
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/tmp/ipykernel_330592/4120876482.py in 
      2 from networkx.algorithms import community
      3 G = nx.petersen_graph()
----> 4 nx_comm.louvain.louvain_communities(G, seed=123)

NameError: name 'nx_comm' is not defined
```

### Expected Behavior
According to the documentation, the output of the above code should be:
`[{0, 4, 5, 7, 9}, {1, 2, 3, 6, 8}]`

### Steps to Reproduce
Just execute the same sample code as described above.

### Environment
Python version: 3.8.10
NetworkX version: 3.0

### Additional context
N/A

**Comment by pauldubois98** [comment]
Yes, this https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.community.louvain.louvain_communities.html#louvain-communities is indeed the correct link.

**Comment by dschult** [comment]
I agree. The import statements should be removed and the calling structure should be:
```python
    nx.community.louvain_communities(G, seed=123)
```

**Comment by pauldubois98** [comment]
I tried:
```
import networkx as nx
G = nx.petersen_graph()
nx.community.louvain_communities(G, seed=123)
```
and I had:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_330592/1814667787.py in 
      1 import networkx as nx
      2 G = nx.petersen_graph()
----> 3 nx.community.louvain_communities(G, seed=123)

AttributeError: module 'networkx.algorithms.community' has no attribute 'louvain_communities'
```
So unfortunetly, this didn't sole the problem (at least in my case) [or perhaps I'm doing something wrong?].

**Comment by pauldubois98** [comment]
Oh, right, my bad, I was on 2.5.
I upgraded and everything works now, thanks!

Should we make a PR to fix the documentation?

**Comment by dschult** [comment]
Yes -- I think a PR to fix that example is the next step. :}

**Comment by pauldubois98** [comment]
I just forked, but it seems that the doc-string is already up to date, so perhaps the documentation website just need to be updated?

**Comment by dschult** [comment]
Aha -- yes, the "stable" documentation for 3.0 didn't have this update and the "latest" changes the imports/calling. The will get fixed when v3.1 is released (soon).  

Thanks for reporting this!

**Comment by pauldubois98** [comment]
Great, I'll close the issue.
Thanks a lot for all your support!
